sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -3911,6 +3911,15 @@ object SQLConf {
     .checkValues(ErrorMessageFormat.values.map(_.toString))
     .createWithDefault(ErrorMessageFormat.PRETTY.toString)
 
+  val INCLUDE_PLANS_IN_ERRORS = buildConf("spark.sql.error.includePlans")
```

Comment:
@dtenedor let's revert the changes in SQLConf.scala

Reply:
@gengliangwang apologies for missing that. It is reverted now.